Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -679,17 +750,51 @@ namespace gdjs {
           brokerServerConfig.port,
           brokerServerConfig.path,
           brokerServerConfig.key,
-          brokerServerConfig.secure
+          brokerServerConfig.secure,
+          { onPeerUnavailable: () => onPeerUnavailable(runtimeScene) }
         );
       } else {
-        gdjs.multiplayerPeerJsHelper.useDefaultBrokerServer();
+        gdjs.multiplayerPeerJsHelper.useDefaultBrokerServer({
+          onPeerUnavailable: () => onPeerUnavailable(runtimeScene),
+        });
       }
 
       _connectionId = connectionId;
       playerNumber = positionInLobby;
       // We save the lobbyId here as this is the moment when the player is really connected to the lobby.
       _lobbyId = lobbyId;
 
+      if (_shouldOpenLobbyPageRightAfterJoiningLobby) {
```

Comment:
Should these mutually exclusive booleans be represented as a single "_actionAfterJoiningLobby" that can be "OPEN_LOBBY_PAGE" | "JOIN_GAME" | "START_GAME"?

Reply:
Good idea, that will be cleaner, changing it